portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches stats from the servers and adds them to the DOM.
+ */
+function generateDiscussion() {
+  	console.log('Fetching discussion.');
+
+    // fetch() function returns a promise
+    const discussionLogPromise = fetch('/data');
+
+	// When the request is complete, pass the response into parseJSON()
+    discussionLogPromise.then(parseJSON);
+}
+
+
+// This function will handle the response and turn it to json
+function parseJSON(response){
+    // TODO: Parse JSON Received from response
+}
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
+	const liElement = document.createElement('li');
```

Comment:
It seems like there is a lot of extra spaces in the code, let's see if their is a way for automatic formatting with the IDE?

Reply:
I've looked into it and it doesn't look like the IDE has autoformatting. But I have looked into Google's style guide and manually formatted it